ntbs-service/Pages/ClinicalTimelines/Edit.cshtml.cs

```diff
@@ -73,7 +73,7 @@ public async Task<IActionResult> OnPostAsync(int? id)
             var notification = await service.GetNotificationAsync(id);
             await service.UpdateTimelineAsync(notification, ClinicalTimeline);
 
-            return RedirectToPage("/Patients/Index");
+            return RedirectToPage("/PreviousHistoryDetails/Edit", new {id = notification.NotificationId});

```

Comment:
We should using Notifications/Edit/... structure, master should be up to date with the latest structure

Reply:
Sure, I'll rebase my two branches onto master